### PR TITLE
ME17: AI85: ME20: Synced SVD files to Register Files.

### PR DESCRIPTION
**Pull request by sihyung-maxim**
AI85 and ME20 parts were built up from the ME17, so those two parts are dependent on some of the ME17-dependent SVD files.
Changes:
- Synced peripheral SVD files to the respective register file.
- Updated max32655.svd, max78000.svd, and max32680.svd.
- Updated copyright notices.
- Resolve SVDConv errors.
- Removed unsupported PWM mode from WUT peripheral.
- Fields that are 1 bit wide should not have two enumerated values, so they've been removed if hand added. This goes for all register files for all parts.

### CODE REVIEW

**Comment by sihyung-maxim** [comment]
I fixed the build errors in the AI85 CNN-related examples. The GCR_PCLKDIV.CNNCLKSEL[17:17] field had a manually coded-in enum definition called MXC_S_GCR_PCLKDIV_CNNCLKSEL_PCLK which wasn't added by the register generation scripts since they don't add enum definitions for fields that are 1 bit wide (only two possible enum value options). This is to prevent register files from being too long and unwieldy, especially for files like gcr_regs.h where there are many registers and fields to define.

Any thoughts on keeping MXC_S_GCR_PCLKDIV_CNNCLKSEL_PCLK defined? It's not a good long term solution to keep adding this definition by hand.

File: Libraries/CMSIS/Device/Maxim/MAX32655/Include/wut_regs.h
Comment (Jacob-Scheiffler) [inline Libraries/CMSIS/Device/Maxim/MAX32655/Include/wut_regs.h:165]: Modes 2-7 are marked as reserved in the UG.

**Review by Jacob-Scheiffler — COMMENTED** [review]
My comments on the MAX32655 WUT regs also go for the MAX32680 and MAX78000 WUT regs.

**Review by lorne-maxim — APPROVED** [review]

File: Libraries/CMSIS/Device/Maxim/MAX32655/Include/wut_regs.h
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32655/Include/wut_regs.h:215]: They should be included, at least SNAPSHOT for the Trim functions.

File: Libraries/CMSIS/Device/Maxim/MAX32655/Include/wut_regs.h
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32655/Include/wut_regs.h]: Removed.

**Review by lorne-maxim — APPROVED** [review]

File: Examples/MAX78000/CNN/UNet-demo/main.c
Comment (sihyung-maxim) [reply Examples/MAX78000/CNN/UNet-demo/main.c:526]: Talked to Jake about this. This might be something that was manually added. Still investigating.

File: Libraries/CMSIS/Device/Maxim/MAX78000/Include/bbfc_regs.h
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX78000/Include/bbfc_regs.h:44]: @Jake-Carter The BBFC is just a deprecated name to the GCFR, and this bbfc_regs.h file just points to gcfr_regs.h. It has been out for a year now, so there's been plenty of grace time to use the GCFR name. The CNN examples are the only files that use bbfc_regs.h. Do you think this should still be included?

File: Libraries/CMSIS/Device/Maxim/MAX32655/Include/wut_regs.h
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32655/Include/wut_regs.h:164]: Actually, I left all the registers and modes in (except for PWM). The Bluetooth Libraries use the other modes.

**Comment by EdwinFairchild** [comment]
This will keep failing because the branch was made before a certain fix was implemented in the workflow that caused a false failed test. You can ignore the failure, or merge main into this and it will pass.

File: Libraries/CMSIS/Device/Maxim/MAX78000/Include/bbfc_regs.h
Comment (Jake-Carter) [reply Libraries/CMSIS/Device/Maxim/MAX78000/Include/bbfc_regs.h:44]: @sihyung-maxim Yes, we should still keep this file around.  New projects can easily be updated to use the new definition, but we should not intentionally break backwards compatibility in any way and there's no maintenance overhead for this file.

If it actually had some conflicting register definitions in it it'd be a different story, but it's just a passthrough file.

File: Examples/MAX78000/CNN/UNet-demo/main.c
Comment (Jake-Carter) [reply Examples/MAX78000/CNN/UNet-demo/main.c:526]: Found it:  
* https://github.com/MaximIntegratedAI/ai8x-synthesis/blob/dd91220a323f69a6e2016fc6a0419ea58d802c92/izer/toplevel.py#L547

Another backwards compatibility issue...  The izer has been using this for a very long time and the old register value currently exists in `gcr_regs.h`:  https://github.com/Analog-Devices-MSDK/msdk/blob/f1301b69c15c8ef34508ebc7b1870190ea4ca5ac/Libraries/CMSIS/Device/Maxim/MAX78000/Include/gcr_regs.h#L391

We can update the izer and examples to use the new definition, but we should not break backwards compatibility by removing `MXC_S_GCR_PCLKDIV_CNNCLKSEL_PCLK`

**Review by Jacob-Scheiffler — DISMISSED** [review]

**Review by Jake-Carter — CHANGES_REQUESTED** [review]
Should resolve a few of the backwards compatibility issues before merging

File: Examples/MAX78000/CNN/UNet-demo/main.c
Comment (sihyung-maxim) [reply Examples/MAX78000/CNN/UNet-demo/main.c:526]: MXC_S_GCR_PCLKDIV_CNNCLKSEL_PCLK will be something that has to be manually added if we want to keep this definition. Is there a CNN header file where we can add this definition so as to not have to worry about adding them by hand each time the register files are generated?

File: Libraries/CMSIS/Device/Maxim/MAX78000/Include/bbfc_regs.h
Comment (sihyung-maxim) [inline Libraries/CMSIS/Device/Maxim/MAX78000/Include/bbfc_regs.h:5]: Adding this note as a unique token to exclude checking these registers files for the new Register_SVD workflow. Let me know if you think there's a better way to handle this.

File: Libraries/CMSIS/Device/Maxim/MAX78000/Include/bbfc_regs.h
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX78000/Include/bbfc_regs.h:44]: I added it back in with some updates.

**Review by lorne-maxim — APPROVED** [review]

**Comment by Jake-Carter** [comment]
@sihyung-maxim looks good, I just want to run a quick regression test of the izer output against these changes.  Will do tomorrow and approve if there are no build issues

**Review by Jake-Carter — APPROVED** [review]
Approved - latest izer output for AI85 builds cleanly for all projects.

FYI only the includes below are used in the generated main.c files.  Looks like the other inclusions (`gcfr_regs.h`) are manual additions for a few demo-type projects
```C
 #include <stdlib.h>
 #include <stdint.h>
 #include <string.h>
 #include <stdio.h>
 #include "mxc.h"
 #include "cnn.h"
 #include "sampledata.h"
 #include "sampleoutput.h"
```

**Pull request merged**
